graviti/openapi/search.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+#
+# Copyright 2022 Graviti. Licensed under MIT License.
+#
+
+"""Interfaces about the search."""
+
+from typing import Any, Dict
+from urllib.parse import urljoin
+
+from graviti.openapi.requests import open_api_do
+
+
+def create_search(
+    access_key: str,
+    url: str,
+    owner: str,
+    dataset: str,
+    *,
+    commit_id: str,
+    sheet: str,
+    criteria: Dict[str, Any],
+    offset: int = 0,
+    limit: int = 128,
+) -> Dict[str, Any]:
+    """Execute the OpenAPI `POST /v2/datasets/{owner}/{dataset}/commits/{commit_id}/sheets\
+    /{sheet}/search`.
+
+    Arguments:
+        access_key: User's access key.
+        url: The URL of the graviti website.
+        owner: The owner of the dataset.
+        dataset: Name of the dataset, unique for a user.
+        commit_id: The commit id.
+        sheet: The sheet name.
+        criteria: The criteria of the search.
+        offset: The offset of the page.
+        limit: The limit of the page.
+
+    Returns:
+        The response of OpenAPI.
+
+    Examples:
+        >>> create_search(
+        ...     "ACCESSKEY-********",
+        ...     "https://api.graviti.com/",
+        ...     "czhual",
+        ...     "MNIST",
+        ...     commit_id = "fde63f357daf46088639e9f57fd81cad",
+        ...     sheet = "train",
+        ...     criteria = {
+        ...         "opt": "or",
+        ...         "value": [
+        ...             {
+        ...                 "opt": "eq",
+        ...                 "key": "rider.uid",
+        ...                 "value": "37574579559858703"
+        ...             },
+        ...             {
+        ...                 "opt": "and",
+        ...                 "value": [
+        ...                     {
+        ...                         "opt": "gt",
+        ...                         "key": "ts",
+        ...                         "value": 1649924877
+        ...                     },
+        ...                     {
+        ...                         "opt": "lt",
+        ...                         "key": "ts",
+        ...                         "value": 2649924877
+        ...                     }
+        ...                 ]
+        ...             }
+        ...         ]
+        ...     }
+        ... )
+        {
+            "data": [
+                {
+                    "__record_key": "123750493121329585",
+                    "ts": 1649924877,
```

Comment:
what is `ts`？

Reply:
just for example, use another column name for better understand